src/components/UI/ReleaseComponentLabel.js

```diff
@@ -91,15 +54,19 @@ const ReleaseComponentLabel = (props) => {
   const { name, version, oldVersion, isAdded, isRemoved, className } = props;
 
   return (
-    <Wrapper className={className}>
-      <NameWrapper>{name}</NameWrapper>
-      <VersionLabel
-        isAdded={isAdded}
-        isRemoved={isRemoved}
-        newVersion={version}
-        oldVersion={oldVersion}
-      />
-    </Wrapper>
+    <ValueLabel
+      className={className}
+      color='#395b70'
```

Comment:
Please use the theme to fetch this color.

You can use this hook to get the theme in a non-styled component:
https://emotion.sh/docs/theming#usetheme-hook